dist/formats/generic/frontend/schema.json

```diff
@@ -164,7 +156,6 @@
         "smart_answer",
         "social_media_use",
         "special_route",
-        "specialist_document",
```

Comment:
these definitely are a thing, what document type covers our specialist documents?

Reply:
`specialist_document` is the `schema_name`, not the `document_type`. Different types of specialist docs have different document types: `aaib_report`, `maib_report`, `service_standard_report`, etc.

Reply:
that's always confused me, thanks